src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double width;
+    private final double length;
+    private String color;
```

Comment:
Think about which variables should be local in the method and which should be class-level. Since 'color' is a property of the figure and is set in the constructor, it should be a final field, to ensure the immutability of Rectangle instances.